src/components/progressReportBreadcrumb/ProgressReportBreadcrumb.tsx

```diff
@@ -0,0 +1,20 @@
+import { Skeleton } from '@mui/material';
+import { Nullable } from '~/common';
+import { ProgressReportDetailFragment } from '~/scenes/ProgressReports/Detail/ProgressReportDetail.graphql';
+import { Breadcrumb } from '../Breadcrumb';
+import { idForUrl } from '../Changeset';
+import { ReportLabel } from '../PeriodicReports/ReportLabel';
+
+interface ProgressReportBreadcrumbProps {
+  data?: Nullable<ProgressReportDetailFragment>;
```

Comment:
You've got the fragment declared right, you just need to use it here. Using the breadcrumb fragment doesn't cross boundaries like the ProgressReportDetail fragment does. That fragment is specific to a view in the scenes. This component should not care about that. Nothing outside of scenes/ should ever import from scenes/. This is why we make our own fragment here.
```suggestion
  data?: Nullable<ProgressReportBreadcrumbFragment>;
```

Reply:
Wow, I need to pay closer attention to what intenseness is importing.  I thought I had the ProgressReportBreadcrumbFragment used here.  I trusted auto fill to quickly.